packages/ove-asset-manager/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@ove/ove-asset-manager",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "npx rimraf dist && npx rimraf src/OVE.Service.AssetManager/bin && npx rimraf src/OVE.Service.AssetManager/obj && npx rimraf AssetFiles.db",
+    "build": "dotnet restore src/OVE.Service.AssetManager && dotnet build src/OVE.Service.AssetManager && dotnet publish src/OVE.Service.AssetManager -o ../../dist",
+    "start": "dotnet dist/OVE.Service.AssetManager.dll --server.urls=http://0.0.0.0:${PORT-'8080'}"

```

Comment:
You should remove default port of 8080, otherwise this may cause conflict with a missing config

Reply:
For example, refusing to start if a port is not defined would be superior.
This would be ok if everything ran on its own container, however, since it is in the services, this can be presumed to not be the case

Reply:
removed